aws/resource_aws_dms_certificate.go

```diff
@@ -43,6 +44,7 @@ func resourceAwsDmsCertificate() *schema.Resource {
 				ForceNew:  true,
 				Sensitive: true,
 			},
+			"tags": tagsSchemaForceNew(),
```

Comment:
Hi @snowsky, this argument is unique in that it can actually be updated for the resource, while it looks like the other resource arguments trigger resource re-creation. 

with that said: let's update this to use `tagsSchema()` instead

Reply:
also could you add this argument to the resource's documentation page?

Reply:
Thanks @anGie44 for the details. To update the certificate resource, do we need a new struct call "ModifyCertificateInput"?

Reply:
sure thing @snowsky! To do an update, since it only affects the resource's `tags` at the moment (in the future, perhaps we'll make use of the `ModifyCertificate` API method to update other resource settings), we don't need to call the Modify API method, but rather use the `keyvaluetags` package's Update method specific to the DMS service e.g.
```go
if d.HasChange("tags") {
		arn := d.Get("certificate_arn").(string)
		o, n := d.GetChange("tags")

		if err := keyvaluetags.DatabasemigrationserviceUpdateTags(conn, arn, o, n); err != nil {
			return fmt.Errorf("error updating DMS Certificate (%s) tags: %w", arn, err)
		}
	}
```

Reply:
Thanks @anGie44, the new changes should be good now. 👍